bedrock/firefox/urls.py

```diff
@@ -61,7 +63,12 @@
         views.FeaturesPrivateBrowsingView.as_view(),
         name='firefox.features.private-browsing'),
     url(r'^firefox/ios/testflight/$', views.ios_testflight, name='firefox.ios.testflight'),
-    page('firefox/mobile', 'firefox/mobile/index.html'),
+
+    url('^firefox/mobile',
```

Comment:
The reason the redirect tests are failing is because it expects this URL to end in a slash. The `page()` helper adds it for you, but here you have to add it yourself.

```suggestion
    url('^firefox/mobile/$',
```

Reply:
ooooh, thank you so much @pmac 👍